IntegrationTests/tests_04_performance/test_01_resources/run-nio-alloc-counter-tests.sh

```diff
@@ -18,25 +18,24 @@ here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 tmp_dir="/tmp"
 
-function die() {
-    echo >&2 "ERROR: $*"
-    exit 1
-}
-
 while getopts "t:" opt; do
     case "$opt" in
         t)
             tmp_dir="$OPTARG"
             ;;
-        \?)
-            die "unknown option $opt"
+        *)
+            exit 1
             ;;
     esac
 done
 
+shift $((OPTIND-1))
+
+test_pattern="${1:-test_*.swift}"
```

Comment:
I think a better way to do this (which also doesn't break on newlines) is to start with

    tests_to_run=( "$here"/test_*.swift )

and if we only run one test, then we can override this with all the arguments that we were given:

    tests_to_run=$@

then, below we can use ...

Reply:
Yeah great suggestion. I only handled specifying a single test argument, as that was the use case I kept running into. But there's no reason not to allow multiple tests to be specified, and this is a good implementation for that.

I'll update!

Reply:
Ah I think I might have misunderstood the intention here. Your suggestion will still only run one test right? It will just treat all of the remaining arguments as a single test argument?